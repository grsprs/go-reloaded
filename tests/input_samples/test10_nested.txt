1F (hex) (up) becomes uppercase number