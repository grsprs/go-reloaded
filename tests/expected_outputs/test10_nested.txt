31 becomes uppercase number